go.mod

```diff
@@ -3,15 +3,16 @@ module cloud.redhat.com/clowder/v2
 go 1.13
 
 require (
-	github.com/go-logr/logr v0.1.0
+	github.com/go-logr/logr v0.2.1
```

Comment:
I'd prefer not to update all these mods in this PR, same as the one below, unless there is a good reason for this?

Reply:
Yes, I had a separate commit that explained the reason but it got rolled into a rebase: I had to update k8s packages to 0.19.3 to pick up `metav1.Condition`

https://github.com/RedHatInsights/cyndi-operator/blob/master/api/v1alpha1/cyndipipeline_types.go#L79